build.sbt

```diff
@@ -12,6 +12,7 @@ libraryDependencies ++= Seq(
 scriptedSettings
 
 enablePlugins(GitVersioning)
+git.versionProperty := "version"
```

Comment:
I can remove this; it was erroneously added. Had this for local testing against other repos so I could set a custom version when publishing locally.